README.md

```diff
@@ -1 +1,55 @@
-# rivet-jekyll-theme
\ No newline at end of file
+[//]: # "Copyright (C) 2019 The Trustees of Indiana University"
+[//]: # "SPDX-License-Identifier: BSD-3-Clause"
+
+# rivet-jekyll-theme
+
+This repository holds the code for a [Rivet](http://rivet.iu.edu)/[Rivet Shell](https://rivet.iu.edu/add-ons/rivet-shell/) theme for use in GitHub Pages or Jekyll.
+
+## Use the theme:
+
+### 1. Gemfile
+
+Add the following to your Gemfile in your Jekyll site:
+    
+```
+gem "rivet-jekyll-theme", :git => "git://github.com/indiana-university/rivet-jekyll-theme.git"
+```
+
+### 2. _config.yml
+
+In your `_config.yml` file, replace the `theme` property value with `rivet-jekyll-theme`:
+
+```
+theme: rivet-jekyll-theme
+```
+
+### 3. Bundle
+
+You'll also need to run `bundle install` to download the theme like adding any Gem.
```

Comment:
Should we add `gem install bundler` before the `bundle install`? Only reason I ask is because I had to do this when setting up locally.

Reply:
IDK. I'm a Ruby noob so how it all fits together and exactly what all those commands do is a little bit of a mystery TBH.